go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/ava-labs/awm-relayer
 go 1.18
 
 require (
-	github.com/ava-labs/avalanchego v1.10.14
-	github.com/ava-labs/subnet-evm v0.5.8
-	github.com/ava-labs/teleporter v0.0.0-20231107202133-eea4d6312c35
+	github.com/ava-labs/avalanche-network-runner v1.7.3-0.20231026153158-2931f8a448d9
```

Comment:
For my understanding, why/where did the Avalanche Network Runner dependency come from?

Reply:
This accidentally got promoted to a direct dependency by some test files that weren't committed. Corrected now.